api/database/functions.py

```diff
@@ -48,8 +53,21 @@ async def execute_sql(sql, param={}, debug=False, engine_type=EngineType.PLAYERD
             records = sql_cursor(rows) if has_return else None
             # commit session
             await session.commit()
+            await session.close()
```

Comment:
`async with engine.session() as session:` handles the open and close of a session